packages/lab/src/layout/LayerLayout/LayerLayout.css

```diff
@@ -0,0 +1,112 @@
+.uitkLayerLayout {
+  --layer-border-color: var(--uitkLayerLayout-border-color, var(--uitk-container-border-color-medium));
+  --layer-box-shadow: var(--uitkLayerLayout-box-shadow, var(--uitk-overlayable-shadow-modal));
+  --layer-background-color: var(--uitkLayerLayout-background-color, var(--uitk-container-background-medium));
+  --layer-border-width: var(--uitkLayerLayout-border-color, var(--uitk-container-border-width));
+  --layer-padding: var(--uitkLayerLayout-padding, calc(3 * var(--uitk-size-unit)));
+  --layer-border-style: var(--uitkLayerLayout-border-style, var(--uitk-container-border-style));
```

Comment:
we decided on using camel case for attributes in tokens, so e.g. --layer-borderStyle and --uitkLayerLayout-borderStyle. most components still have the old style but ready ones (e.g. Button) have adopted the camel case already which can be looked at for reference :) 

also --uitk-container-border-style uses camel case now as well as border width/color above.. perhaps it was missed in a rebase here? best to check where ever the characteristics are used

Reply:
Makes sense, I've updated these